bin/metrics-docker-stats.rb

```diff
@@ -160,9 +160,9 @@ def list_containers
 
     containers.each do |container|
       list << if config[:friendly_names]
-                container['Names'][0].delete('/')
+                container['Names'][-1].delete('/')
```

Comment:
should we delete the the `/` or remove everything after the `/`? I think the reason that the `/` was being removed was because its not a valid character according to the [source code](https://github.com/moby/moby/blob/8e610b2b55bfd1bfa9436ab110d311f5e8a74dcb/daemon/names/names.go#L5-L6) in a name.

Reply:
True, the character is not a valid character.

Reply:
in case multiple containers are linked together the container name is concatenate with the other container seperate by `/` .

`./embedded/bin/metrics-docker-stats.rb -n true`

### ACTUAL RESULT

```
Check failed to run: 404 "Not Found", ["/opt/sensu/embedded/lib/ruby/2.4.0/net/http/response.rb:122:in `error!'", "/opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/sensu-plugins-docker-3.1.1/lib/sensu-plugins-docker/client_helpers.rb:41:in `call'", "/opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/sensu-plugins-docker-3.1.1/bin/metrics-docker-stats.rb:175:in `container_stats'", "/opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/sensu-plugins-docker-3.1.1/bin/metrics-docker-stats.rb:122:in `block in run'", "/opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/sensu-plugins-docker-3.1.1/bin/metrics-docker-stats.rb:121:in `each'", "/opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/sensu-plugins-docker-3.1.1/bin/metrics-docker-stats.rb:121:in `run'", "/opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/sensu-plugin-2.7.0/lib/sensu-plugin/cli.rb:57:in `block in <class:CLI>'"]
```

### EXPECTED RESULT

```
www-prod-do-sgp1-42.docker.container-xxx-prod.name container-xxx-prod 1542816915
```